src/emmy/function.cljc

```diff
@@ -12,7 +12,8 @@
             [clojure.core.match :refer [match]]
             [emmy.generic :as g]
             [emmy.util :as u]
-            [emmy.value :as v])
+            [emmy.value :as v]
+            [taoensso.timbre :as log])
```

Comment:
Remove this if it's not needed

Reply:
I was using it but have refactored things now to avoid it. This PR has been through a few layers that I'm sanding out. Initially I tried to dodge the fn-with-meta problem by making functions with "manifest arity" in this case and letting the arity cache deal with it in the future, but now we hack the arity into the function objects at creation time which is easier to reason about IMO